### PR TITLE
48035_add_activities_for_trainne_by_public_activity

### PR DESCRIPTION
**Pull request by zenith19**

### CODE REVIEW

File: app/controllers/supervisor/courses_controller.rb [hunk 1]
```diff
@@ -24,7 +24,7 @@ def create
 
   def show
     @activities = PublicActivity::Activity.order("created_at desc")
-      .where(trackable_type: "Course", recipient_id: @course.id)
+      .where(trackable_type: "Course", trackable_id: @course.id)
```
Comment (abdullahallmehedi) [inline app/controllers/supervisor/courses_controller.rb]: ()

File: app/controllers/supervisor/subjects_controller.rb [hunk 2]
```diff
@@ -10,7 +10,7 @@ def show
     @subject = Subject.find params[:id]
     @tasks = @subject.tasks
     @subject_activities = PublicActivity::Activity.order("created_at desc").
-      where(trackable_type: "Subject", recipient_id: @subject.id)
+      where(trackable_type: "Subject", trackable_id: @subject.id)
```
Comment (abdullahallmehedi) [inline app/controllers/supervisor/subjects_controller.rb]: ()

**Comment by abdullahallmehedi** [comment]
ok

**Comment by srahmanbd** [comment]
Mehedi, Great!!!!
Zenith, Great!!!! Next time you won't have this easy misses we believe.

**Pull request merged**